realm/src/androidTest/java/io/realm/RealmLinkTests.java

```diff
@@ -463,4 +463,29 @@ public void testSubquery() {
         RealmResults<Owner> subOwners = owners.where().equalTo("cat.name", "Blackie").findAll();
         assertEquals(1, subOwners.size());
     }
+
+    public void testIsNull() {
+        RealmResults<Owner> owners1 = testRealm.where(Owner.class).isNull("dogs").findAll();
```

Comment:
When we get support for null, I would like to extend this predicate to a more general one.